packages/api/amplify_api/test/amplify_api_config_test.dart

```diff
@@ -0,0 +1,84 @@
+import 'dart:convert';
+
+import 'package:amplify_api/src/amplify_api_config.dart';
+import 'package:amplify_core/amplify_core.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'test_data/fake_amplify_configuration.dart';
+
+void main() {
+  late EndpointConfig endpointConfig;
+
+  group('GraphQL Config', () {
+    const endpointType = EndpointType.graphQL;
+    const endpoint =
+        'https=//abc123.appsync-api.us-east-1.amazonaws.com/graphql';
```

Comment:
nit: could you please make the first part "https://" so it's more like a normal URI instead of with "="?